### PR TITLE
fix members description form MSON

### LINKED ISSUE #96: MSON - array|enum - members description
## input
```
# Example

## GET /
+ Attributes

    + id: 21 (enum[number], optional) - Database ID
        + Members
            + 37 (number) - Testing value
            + 23 - should be number because of `enum[number]`
            
+ Response 201
```

## response (partial)
```
....
                    "value": {
                              "element": "enum",                                                   
                              "content": [                                                         
                                { 
                                  "element": "number",                                             
                                  "content": 21                                                    
                                },                                                                 
                                { 
                                  "element": "number",                                             
                                  "content": 37                                                    
                                },                                                                 
                                { 
                                  "element": "number",                                             
                                  "content": 23                                                    
                                }
```

## bug
- elements with values 23 && 37 should contain `meta.description` for blueprint above

### PR DESCRIPTION
**Pull request by klokane**
This patch close #96 **MSON - array|enum - members description**

### CODE REVIEW

File: src/RefractDataStructure.cc [hunk 1]
```diff
@@ -502,18 +506,33 @@ namespace drafter {
...
+        Join join(descriptionRef);
+
+        refract::IElement::MemberElementCollection::iterator iterator = value->meta.find(SerializeKey::Description);
+        if (iterator != value->meta.end()) {
+            // There is already setted description to "value" as result of `RefractElementFromValue()`
+            // so we need to move this atribute from "value" up to MemberElement
+            //
+            // NOTE: potentionaly unsafe, but we set it already to StringElement
+            // most safe is check it via refract::TypeQueryVisitor
+            descriptionRef = (static_cast<refract::StringElement*>((*iterator)->value.second)->value);
```
Comment (pksunkara) [inline src/RefractDataStructure.cc]: Why are we not using `refract::TypeQueryVisitor`?

**Comment by klokane** [comment]
sry missing tests will add

**Comment by pksunkara** [comment]
Please add the test case for this.

File: src/RefractDataStructure.cc (hunk 1 above)
Comment (klokane) [inline src/RefractDataStructure.cc]: It is explained in comment

**Comment by klokane** [comment]
Test case for member description added

**Pull request merged**